DFE-Digital/Dfe.Academies.External
# Added current financial year page and refactored the ConversionApplic…

…ationRetrievalService.cs to use a a <string,dynamic> dictionary for properties to update on a ConversionApplication to reduce code duplication. Also fixed an issue with the IDs not being unique on the radio buttons so selecting a radio button would change the value in another pair on the page.


Resolves #<1234>

## Type of change


## Expected behaviour


The current financial page has been added. Filling the form in should record all the data on the fields.

Changing the Surplus or Deficit radio button values on either the Previous or Current financial year pages are linked because the IDs weren't unique. So changing the revenue radio button would change the capital revenue button


## Steps to reproduce issue (if relevant)


1. Go to the previous or current financial page
2. Fill both pairs of radio buttons
3. Change one of them and see if the other changes too
4.

## Steps to test this PR

1. Fill out the current financial year page form
2. See if the data is recorded in the database and gets pulled back in the UI when reloading
3.
4.

## Prerequisites
